src/libs/protobuf_clips/communicator.h

```diff
@@ -217,7 +217,7 @@ class ClipsProtobufCommunicator
 	  sig_peer_sent_;
 
 	fawkes::Mutex map_mutex_;
-	long int      next_client_id_;
+	long int      next_client_id_ = 0;
```

Comment:
For consistency's sake, this should be initialized in the constructor(s), not in the header. All the other members are also initialized in the constructors.